src/components/navigation/SideMenuButton.tsx

```diff
@@ -12,6 +13,7 @@ export const SideMenuButton = () => {
   const theme = useTheme()
 
   const handlePress = useHandler(() => {
+    Keyboard.dismiss()
```

Comment:
This is fine, but it doesn't account for the swipe-to-open gesture.

You can use the [useDrawerStatus](https://reactnavigation.org/docs/7.x/drawer-navigator/#usedrawerstatus) hook inside the drawer itself:

```ts
const isOpen = useDrawerStatus() === 'open'
React.useEffect(()=> {
  if (isOpen) Keyboard.dismiss()
}, [isOpen])
```

That should work for both button press & swiping.

Reply:
Actually, no, this is impossible. Swiping to open the drawer dismisses the keyboard.